couchpotato/core/downloaders/rtorrent_.py

```diff
@@ -156,12 +156,13 @@ def download(self, data = None, media = None, filedata = None):
 
         # Try download magnet torrents
         if data.get('protocol') == 'torrent_magnet':
-            filedata = self.magnetToTorrent(data.get('url'))
-
-            if filedata is False:
-                return False
-
-            data['protocol'] = 'torrent'
+            magnet_url = data.get('url')
+            # Pull out the ID of the torrent
+            urn_btih_position = magnet_url.index('urn:btih:')
+            torrent_hash = magnet_url[urn_btih_position + 9:urn_btih_position+49]
+            # Send magnet URL directly to rTorrent
+            self.rt.load_torrent_simple(magnet_url, 'url', start=True)
+            return torrent_hash
```

Comment:
It should return a download id of the downloader. like below `self.downloadReturnId(torrent_hash)`